cmd/runtimetest/main.go

```diff
@@ -191,6 +191,20 @@ func validateSysctls(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateMaskedPaths(spec *rspec.Spec) error {
+	fmt.Println("validating maskedPaths")
+	for _, v := range spec.Linux.MaskedPaths {
+		fi, err := os.Stat(v)
+		if err != nil {
+			return err
+		}
+		if fi.Mode()&0444 != 0 {
+			return fmt.Errorf("%v should be readonly", v)
```

Comment:
The error message shouldn't be about “readonly”.  And this is probably another place where we want to use [“just try it”][1] (in this case, read and write) instead of a mode check.

[1]: https://github.com/opencontainers/ocitools/pull/78#issuecomment-221837970